packages/app-project/src/components/ProjectHeader/helpers/getNavLinks/getNavLinks.js

```diff
@@ -3,26 +3,31 @@ import counterpart from 'counterpart'
 function getNavLinks (isLoggedIn, baseUrl) {
   const links = [
     {
-      href: `${baseUrl}/about`,
+      as: `${baseUrl}/about`,
+      href: `/projects/[owner]/[project]/about`,
```

Comment:
Maybe link these to PFE for now, until these pages exist? Otherwise, I'm seeing infinite reloading when browsing to these pages.

Reply:
Yeah - so the idea for these was to have a stub for them to navigate to, which then triggers a reload so the route gets picked up by PFE. On the staging environment (if my memory isn't failing me) we have a slightly different CloudFront routing config, so they don't actually fall back.

I'll take a different tack - if a link props object has `as` and `href`, we use `Link`; if it's only `href`, I'll use a regular `a`.

Reply:
Sounds good. I was thinking maybe use full URLs, with the protocol, for external links, and just paths for links that can be handled by the client.